packages/plugin-core/src/showcase/MeetingNotesTip.ts

```diff
@@ -0,0 +1,59 @@
+import { assertUnreachable } from "@dendronhq/common-all";
+import { SegmentClient } from "@dendronhq/common-server";
+import { ShowcaseEntry } from "@dendronhq/engine-server";
+import {
+  MEETING_NOTE_FEATURE_SHOWCASE_TEST,
+  MeetingNoteTestGroups,
+} from "../abTests";
+import { AnalyticsUtils } from "../utils/analytics";
+import {
+  IFeatureShowcaseMessage,
+  DisplayLocation,
+} from "./IFeatureShowcaseMessage";
+
+export class MeetingNotesTip implements IFeatureShowcaseMessage {
+  shouldShow(displayLocation: DisplayLocation): boolean {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage: {
+        const ABUserGroup = MEETING_NOTE_FEATURE_SHOWCASE_TEST.getUserGroup(
+          SegmentClient.instance().anonymousId
+        );
+
+        return (
+          ABUserGroup === MeetingNoteTestGroups.show &&
+          !AnalyticsUtils.isFirstWeek()
+        );
+      }
+      case DisplayLocation.TipOfTheDayView:
+        return true;
+      default:
+        assertUnreachable(displayLocation);
+    }
+  }
+  get showcaseEntry(): ShowcaseEntry {
+    return ShowcaseEntry.TryMeetingNotes;
+  }
+
+  getDisplayMessage(displayLocation: DisplayLocation): string {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage:
+        return `Dendron now has meeting notes. Try it out!`;
```

Comment:
the `Dendron now has...` prefix shouldn't be used if its a tip. a tip generally lets people know about existing functionality. can e change the text to remove indicator of recency?

Reply:
This is carry over from last week's toast experiment.  Given that it was quite effective, I'm inclined to keep the toast message the same, at least for a couple of more weeks.  That toast will not show more than once for users, unless they click "Later".